xhyve/xhyve.go

```diff
@@ -84,6 +85,7 @@ var (
 	ErrMachineNotExist = errors.New("machine does not exist")
 	diskRegexp         = regexp.MustCompile("^/dev/disk([0-9]+)")
 	kernelRegexp       = regexp.MustCompile(`(vmlinu[xz]|bzImage)[\d]*`)
+	kernelOptionRegexp = regexp.MustCompile(`\s*append`)
```

Comment:
> If there is a possibility that append is not indented, no need to change this pattern.

I can't bet of how other distro have it but for Centos/fedora/boot2docker/minikube it's indented I am in favor to keep it as it is in case some distro not follow intended way.